docs/content/configuration/indexing-service.md

```diff
@@ -173,8 +173,8 @@ Issuing a GET request at the same URL will return the current worker config spec
 
 |Property|Description|Default|
 |--------|-----------|-------|
-|`selectStrategy`|How to assign tasks to middle managers. Choices are `fillCapacity`, `fillCapacityWithAffinity`, `equalDistribution` and `javascript`.|fillCapacity|
-|`autoScaler`|Only used if autoscaling is enabled. See below.|null|
+|`worker.config.selectStrategy.type`|How to assign tasks to middle managers. Choices are `fillCapacity`, `fillCapacityWithAffinity`, `equalDistribution` and `javascript`.|fillCapacity|
```

Comment:
The old docs are wrong, but I think the new docs are wrong too. This block is meant to be documenting the JSON form of dynamic worker config, so the property-style "worker.config.selectStrategy.type" string shouldn't apply.

More correct for "selectStrategy" should be something like "this is an object, the options are documented below"